### PR TITLE
feat: added wishlist feature

### PR DESCRIPTION
**Pull request by anupkgurung**
Added add to wishlist 
Remove from wishlist
from product listing and home page.

### CODE REVIEW

**Review by hrishikeshmane — APPROVED** [review]
Looks good to me. Clean and readable code.

File: src/components/Navbar/Navbar.jsx [hunk 1]
```diff
@@ -21,13 +23,13 @@ export const Navbar = () =>{
                             <span className="material-icons pl-1">account_circle</span>
                         </li>
                         <li className="list-inline">
-                            <Link className="link-style-none" to="/">
-                                {/* href="./wishlist.html */}
```
Comment (ianuragsharma) [inline src/components/Navbar/Navbar.jsx:25]: remove these comments

File: src/components/Product/ProductCard.jsx [hunk 2]
```diff
@@ -1,45 +1,45 @@
...
-                   <div className='card-container card-container--verticle product-card-container'>
-                       <span className="badge text-badge z-1 m-0">{discount}%</span>                         
-                       <div className="card-header">                                
-                           <div className="card-img-container position-rel">
-                               <img className="responsive img-radius position-abs" src={image} alt="Art of Photography" />
-                           </div>
-                           <div className="card-desc m-2 card-desc--overlay">
-                               <span className="price">₹{price}</span> | 
-                              <span className="title">{title}</span>
-                               {/* <span className="author">Héctor García </span>  */}
```
Comment (ianuragsharma) [inline src/components/Product/ProductCard.jsx:32]: remove comments

File: src/components/Navbar/Navbar.jsx [hunk 3]
```diff
@@ -21,13 +23,13 @@ export const Navbar = () =>{
...
+                            <Link className="link-style-none relative" to="/wishlist">
                                 <span className="material-icons pl-1">favorite_border</span>
+                                <span className="badge icon-badge wishlist-badge">{wishlist && wishlist.length}</span>
                             </Link>
                         </li>
                         <li className="list-inline">
-                            <Link className="link-style-none" to="/">
+                            <Link className="link-style-none relative" to="/">
                                 {/* href="./cart.html" */}
```
Comment (ianuragsharma) [inline src/components/Navbar/Navbar.jsx:33]: same here

**Review by ianuragsharma — APPROVED** [review]
good to see that you have destructured everything nicely.
File structure is very well defined too.
Good to merge

**Pull request merged**